k8s/Makefile

```diff
@@ -35,20 +35,13 @@ proxy:
 	open 'http://127.0.0.1:8001/ui'
 	kubectl proxy
 
-rollout:
+patch:
 	-kubectl patch deployment ethereum-deploy -p \
 		$(subst CONTSHA,$(shell ./docker-repodigest ethereum),$(subst CONTNAME,ethereum,$(ROLLOUT_PATCH)))
 	-kubectl patch deployment chainlink-deploy -p \
 		$(subst CONTSHA,$(shell ./docker-repodigest chainlink),$(subst CONTNAME,chainlink,$(ROLLOUT_PATCH)))
 
-rollback:
-	kubectl rollout undo deployment/testnet-deploy
```

Comment:
Are we sure we want to remove `rollback`?

Reply:
I think so. It's a one liner, and it's out of date... Maybe add it back when we start using it more?